src/app/settings/admin/opa/default-constraints/component.ts

```diff
@@ -74,6 +77,13 @@ export class DefaultConstraintComponent implements OnInit, OnChanges, OnDestroy
     });
   }
 
+  ngAfterViewChecked(): void {
+    const box = document.querySelector('.applies-to-column') as HTMLElement;
```

Comment:
This can most likely be achieved with the combination of `ngClass` and status check of the component. Also, using `document` without importing breaks the tree-shaking algorithms. This should be optimized.